javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,164 @@
+// Voice assistant
+
+
+let nameArr = [];
+let getName = "";
+let todo = [];
+const favsportsArr = [];
+
+function getReply(command) 
+{
+
+    if (typeof command !== "string") {
+        return "please enter valid string"
+    }
+    let stringName = command.toLowerCase().split(' '); // This will make all letters into lowercase
+
+    if (command.toLowerCase().includes('hello my name is')) {
```

Comment:
You are using `command.toLowerCase()` multiple times so create a new variable for it.
```suggestion
    const lowerCaseCommand = command.toLowerCase();
    if (lowerCaseCommand.includes('hello my name is')) {
```

Reply:
will keep this useful information for future programming